cmd/osbuild-tests/main_test.go

```diff
@@ -18,24 +18,48 @@ import (
 	"testing"
 )
 
-func TestEverything(t *testing.T) {
-	// Smoke tests, until functionality tested fully
-	// that the calls succeed and return valid output
+func TestComposeCommands(t *testing.T) {
 	runComposerCLI(t, false, "compose", "types")
 	runComposerCLI(t, false, "compose", "status")
 	runComposerCLI(t, false, "compose", "list")
 	runComposerCLI(t, false, "compose", "list", "waiting")
 	runComposerCLI(t, false, "compose", "list", "running")
 	runComposerCLI(t, false, "compose", "list", "finished")
 	runComposerCLI(t, false, "compose", "list", "failed")
-	// runCommand(false, "compose", "log", UUID, [<SIZE>])
-	// runCommand(false, "compose", "cancel", UUID)
-	// runCommand(false, "compose", "delete", UUID)
-	// runCommand(false, "compose", "info", UUID)
-	// runCommand(false, "compose", "metadata", UUID)
-	// runCommand(false, "compose", "logs", UUID)
-	// runCommand(false, "compose", "results", UUID)
-	// runCommand(false, "compose", "image", UUID)
+
+	// Full integration tests
+	uuid := buildCompose(t, "ami")
+	defer deleteCompose(t, uuid)
+
+	runComposerCLI(t, false, "compose", "info", uuid.String())
+
+	runComposerCLI(t, false, "compose", "metadata", uuid.String())
+	_, err := os.Stat(uuid.String() + "-metadata.tar")
+	require.NoError(t, err, "'%s-metadata.tar' not found", uuid.String())
+	defer os.Remove(uuid.String() + "-metadata.tar")
+
+	runComposerCLI(t, false, "compose", "results", uuid.String())
+	_, err = os.Stat(uuid.String() + ".tar")
+	require.NoError(t, err, "'%s.tar' not found", uuid.String())
+	defer os.Remove(uuid.String() + ".tar")
+
+	runComposerCLI(t, false, "compose", "log", uuid.String())
+	runComposerCLI(t, false, "compose", "log", uuid.String(), "1024")
+	//todo: ^^^ verify output
+
+	runComposerCLI(t, false, "compose", "logs", uuid.String())
+	_, err = os.Stat(uuid.String() + "-logs.tar")
+	require.NoError(t, err, "'%s-logs.tar' not found", uuid.String())
+	defer os.Remove(uuid.String() + "-logs.tar")
+
+	// runComposerCLI(t, false, "compose", "cancel", UUID)
+	runComposerCLI(t, false, "compose", "image", uuid.String())
+	_, err = os.Stat(uuid.String() + ".ami")
+	require.NoError(t, err, "'%s.ami' not found", uuid.String())
```

Comment:
@major I see:

```
skipping: [localhost] => (item={'name': 'osbuild-tests', 'timeout': 30}) 
```

and the above piece of code is wrong. The filename is `<uuid>-image.vhdx` now. Do you know why Schutzbot skipped this test ?

Reply:
Nevermind, I found the failure. I've been looking at the output of another job.